releasenotes/notes/parallel-solvers-ae9c2a2681155ada.yaml

```
@@ -0,0 +1,8 @@
+---
+upgrade:
```

This should listed as a `feature`, upgrade in release notes is used to note changes (ie to API) that could affect existing user code.
```suggestion
feature:
```